rules/rollup.go

```diff
@@ -427,24 +456,20 @@ func (rc *rollupRule) revive(
 	if !rc.Tombstoned() {
 		return fmt.Errorf("%s is not tombstoned", n)
 	}
-	return rc.addSnapshot(name, rawFilters, targets, cutoverTime)
+	return rc.addSnapshot(name, rawFilters, targets, meta)
 }
 
-func (rc *rollupRule) history() []RollupRuleView {
-	views := make([]RollupRuleView, len(rc.snapshots))
-	for i, s := range rc.snapshots {
-		views[i] = newRollupRuleView(rc.uuid, *s)
+func (rc *rollupRule) history() []*RollupRuleView {
+	views := make([]*RollupRuleView, len(rc.snapshots))
+	for i := 0; i < len(rc.snapshots); i++ {
+		rrs, _ := newRollupRuleView(rc, i)
+		views[i] = rrs
```

Comment:
If someone made a change, in this history view, every past Rule will have the guy as the LastUpdatedBy, is this what we want?

Reply:
Good point. Its probably better if we do user/timestamp per snapshot. I'll change that.